Description: Install examples to /usr/share/doc/tgt
 By default they get installed to /etc/tgt
Author: Unknown
Forwarded: no

--- a/conf/Makefile
+++ b/conf/Makefile
@@ -11,9 +11,9 @@ install:
 	if [ ! -f $(DESTDIR)$(sysconfdir)/tgt/targets.conf ] ; then \
 		install -m 644 targets.conf $(DESTDIR)$(sysconfdir)/tgt ; \
 	fi
-	install -d -m 755 $(DESTDIR)$(sysconfdir)/tgt/examples
+	install -d -m 755 $(DESTDIR)/usr/share/doc/tgt/examples
 	for f in $(EXAMPLES) ; do \
-		install -m 644 examples/$$f $(DESTDIR)$(sysconfdir)/tgt/examples ;\
+		install -m 644 examples/$$f $(DESTDIR)/usr/share/doc/tgt/examples ;\
 	done
 	install -d $(DESTDIR)$(sysconfdir)/tgt/conf.d
 
